src/app/item-popup/ItemPopupBody.tsx

```diff
@@ -1,91 +0,0 @@
-import RichDestinyText from 'app/dim-ui/destiny-symbols/RichDestinyText';
-import { t } from 'app/i18next-t';
-import { doShowTriage, ItemTriage, TriageTabToggle } from 'app/item-triage/ItemTriage';
-import { percent } from 'app/shell/formatters';
-import clsx from 'clsx';
-import { DimItem } from '../inventory/item-types';
-import { ItemPopupExtraInfo } from './item-popup';
-import ItemDetails from './ItemDetails';
-import styles from './ItemPopupBody.m.scss';
-import './ItemPopupBody.scss';
-
-export const enum ItemPopupTab {
-  Overview,
-  Triage,
-}
-
-/** The main portion of the item popup, with pages of info (Actions, Details, Reviews) */
-export default function ItemPopupBody({
-  item,
-  extraInfo,
-  tab,
-  onTabChanged,
-}: {
-  item: DimItem;
-  extraInfo?: ItemPopupExtraInfo;
-  tab: ItemPopupTab;
-  onTabChanged: (tab: ItemPopupTab) => void;
-}) {
-  const failureStrings = Array.from(extraInfo?.failureStrings || []);
-  if (item.owner !== 'unknown' && !item.canPullFromPostmaster && item.location.inPostmaster) {
-    failureStrings.push(t('MovePopup.CantPullFromPostmaster'));
-  }
-
-  const tabs: {
-    tab: ItemPopupTab;
-    title: JSX.Element | string;
-    component: JSX.Element;
-  }[] = [
-    {
-      tab: ItemPopupTab.Overview,
-      title: t('MovePopup.OverviewTab'),
-      component: <ItemDetails item={item} extraInfo={extraInfo} />,
-    },
-  ];
-  if ($featureFlags.triage && doShowTriage(item)) {
-    tabs.push({
-      tab: ItemPopupTab.Triage,
-      title: <TriageTabToggle item={item} currentTab={tab} />,
-      component: <ItemTriage item={item} />,
-    });
-  }
-
-  return (
-    <div>
-      {/* TODO: Should these be in the details? Or in the header? */}
-      {item.percentComplete !== 0 && !item.complete && (
-        <div className="item-xp-bar" style={{ width: percent(item.percentComplete) }} />
-      )}
```

Comment:
`item-xp-bar` hasn't existed for over four years now as far as I can tell (46a59b2a2dcd27e87f8ca93f2860021373dc2bc8), resulting in an unstyled empty div, which I simply removed. This looks like it was providing a global completion progress bar on the item popup itself, in addition to the bars for every objective...

Reply:
I wonder if that was a D1 thing...